src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/scenario/JiraCoreScenario.kt

```diff
@@ -76,6 +82,7 @@ class JiraCoreScenario constructor() : Scenario {
         val actionProportions = mapOf(
             createIssue to 5,
             searchWithJql to 20,
+            searchWithJqlStar to 6,
```

Comment:
I'm afraid this can be considered as behavioral change and therefore API breaking :frowning:

Reply:
Sure, I have just added this in changelog

Reply:
I agree. Let's not change the behaviour of the default scenario. We can change the behaviour in our custom scenario.